### PR TITLE
Added the contributors.md and a new file in posts.

### PR DESCRIPTION
**Pull request by daniel-plainview**
I do not know how git renames that posts, or is it upto me to do that ,
however the post is short but I hope it makes a difference

### CODE REVIEW

**Review by danielchegwidden — COMMENTED** [review]
amature? Maybe this should be beginner or another synonym as that might confuse people. I can fix any grammar before posting so I wont worry about that here. 

You have put it as a [profiles] but that is not the appropriate category, have a look at the existing categories in the posts and update that. You have also put this as a post when you do hexo new "post" it should be a draft as we don't want posts to automatically go live which will happen if this is merged as is.

You can name your post the same name as the post title for consistency, and I think "the Satellite Link" doesnt really explain what the post is about enought.

What is the purpose of the collaborators file?

**Comment by daniel-plainview** [comment]
1)I shall look into the category
2)The name is a bit murky and doesn’t really convey the whole idea of the post
3)I didn’t  use hexo, but shall look into it nonetheless
4)Whenever a new person contributes to the repo, the contributors is updated , so that the people looking at the repo know who to contact , if they wish to contribute.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Review by danielchegwidden — COMMENTED** [review]
you need to define terms or use more layman terms for things like Agri-Tech and metadata. So that someone who is not a data scientist can understand instantly what you are talking about. We want these posts to be able to attract new data scientists and raise the awareness of data science so think about that audience when using domain-specific works

**Review by danielchegwidden — COMMENTED** [review]
Working at an Agri-Tech (Agricultural Technology) consultancy has taught me a few things about data and drawing inferences from data. The most profound of them is the ability to tell the most useful piece of information from the least. It might happen that the most significant piece of information is actually the metadata. The metadata is a file that is received along with the main file, and contains information regarding describing then data. A good analogy is if the data is the textbook, the metadata is its table of contents.

One of the key technologies being used by the agriculture industry for modelling crops is called Remote Sensing, which is about capturing information about the surface of a planet. My role was to map agricultural fields during various cropping seasons, using a series of satellite images over a time period. In Remote Sensing, the timestamp of the image is a very important piece of data, and it can be found in the metadata, as well as as part of the file name.

One time whilst working with this data, I renamed the file so many times that the timestamp was difficult to retrieve. This is an important piece of data for the analysis, and I had to redo a lot of my work in order to keep the timestamp available. When I spoke about this with my colleague, he pulled up the metadata and showed me where the timestamp could easily be found, independent of the name of the file. This was a hard way to learn the value of the metadata.

So when you want to play around with the data, take a few minutes to glance over the metadata. It might help you answer some simple questions that can guide and inform your later work.

**Comment by daniel-plainview** [comment]
Thank you loads for the changes. This gives me some insight as to what you expect out of a blog. I shall keep these changes in mind for my next post.

**Review by daniel-plainview — COMMENTED** [review]
I have made a few changes, please have a look at them and suggest suitable edits.

File: source/_drafts/The-Importance-of-Metadata.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Working at an  Agri-Tech consultancy has taught me a few things about data and drawing inferences from it. The most profound of them is the ability to tell the most useful piece of information from the least . It might happen that the most significant piece of information is actually the metadata. It is a file that is auxillary to the main file and typically contains information describing the data  that might be helpful in answering rudimentary questions about the contents of the main file.
+
+For example, A census dataset of a province might have a metadata that describes the number of respondents , the type of the data itself (csv,txt) , the source of the dataset (survey year, survey type), the methology of collection(internet survey, ground based survey). A good analogy is if data is the textbook, the metadata is its table of contents.
```
Comment (danielchegwidden) [inline source/_drafts/The-Importance-of-Metadata.md:12]: I dont think this paragraph is relevant, sure it explains metadata more but brining in census data seems to confuse what you are trying to say

File: source/_drafts/The-Importance-of-Metadata.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: The Importance of Metadata
+author: Mukund Srinivas
+date:
+categories: general
+number:
+tags:
+---
+Agricultural technology(Agri-Tech) is a portfolio of products and serivices that help the agricultural industry, these could range from products that help in reducing operational costs of the client(developing algorithms for efficiently moving and storing food), to make policy decisions (What are the crops that need a subsidy that particular season).
```
Comment (danielchegwidden) [inline source/_drafts/The-Importance-of-Metadata.md:9]: Agricultural technology (Agri-Tech) is a portfolio of products and services that help the agricultural industry, ranging from helping in reducing costs (efficiently moving and storing food) to making policy decisions (providing subsidies for crops).

File: source/_drafts/The-Importance-of-Metadata.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: The Importance of Metadata
```
Comment (danielchegwidden) [inline source/_drafts/The-Importance-of-Metadata.md:2]: title: 'Don't forget the importance of metadata'

Comment (daniel-plainview) [reply source/_drafts/The-Importance-of-Metadata.md:2]: Oh yes, That is one nice title

**Comment by danielchegwidden** [comment]
Does a version of this image work?
https://unsplash.com/photos/0xO65I5N6Vw

**Comment by daniel-plainview** [comment]
https://earthobservatory.nasa.gov/images/6605/agricultural-patterns
This is what I was thinking , Shall I make a mosaic of these images ?

**Comment by daniel-plainview** [comment]
I could perhaps add it to the markdown file ?

**Comment by danielchegwidden** [comment]
Do we have permissions to use those? Check out my farming post for what it needs to look like and the code to attach, the image needs to go in the images folder

**Comment by daniel-plainview** [comment]
We would need a folder for images , so that we link them in the markdown, however i do not see a folder called images. Therefore I am creating one in the development branch.

**Review by danielchegwidden — APPROVED** [review]
I will make sure that the image works and fix a few grammar things before posting

**Pull request merged**